<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="register_lbl_create">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>204</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create new User</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_lbl_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_tb_name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn_create">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn_cancel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
